app/views/home/show.html+redesign.erb

```diff
@@ -0,0 +1,180 @@
+<%= open_graph_tags title: t(".title_og"), turbo_native_title: t(".turbo_native_title"), description: t(".description_og") %>
+
+<div class="bg-navy">
+  <div class="flex items-start justify-center max-w-6xl mx-auto px-8 py-48 -mt-24 bg-gradient-radial from-navy-gradient via-navy to-navy sm:-mt-48 sm:pt-72">
```

Comment:
Let's pull this out to a partial. (No VC because I don't think we will re-use it.)

Also, TIL about `bg-gradient-radial`. Nice!